ddtrace/appsec/ddwaf/__init__.py

```diff
@@ -0,0 +1,156 @@
+import ctypes
+import time
+from typing import TYPE_CHECKING
+
+from six import text_type
+
+from ddtrace.internal.logger import get_logger
+
+
+if TYPE_CHECKING:
+    from typing import Any
+    from typing import Union
+
+    from ddtrace.appsec.ddwaf.ddwaf_types import DDWafRulesType
+
+
+LOGGER = get_logger(__name__)
+
+try:
+    from .ddwaf_types import ddwaf_config
+    from .ddwaf_types import ddwaf_context_destroy
+    from .ddwaf_types import ddwaf_context_init
+    from .ddwaf_types import ddwaf_destroy
+    from .ddwaf_types import ddwaf_get_version
+    from .ddwaf_types import ddwaf_init
+    from .ddwaf_types import ddwaf_object
+    from .ddwaf_types import ddwaf_result_free
+    from .ddwaf_types import ddwaf_ruleset_info
+    from .ddwaf_types import ddwaf_update_rule_data
+    from .ddwaf_types import py_ddwaf_required_addresses
+    from .ddwaf_types import py_ddwaf_run
+
+    _DDWAF_LOADED = True
+except OSError:
+    _DDWAF_LOADED = False
+    LOGGER.warning("DDWaf features disabled. WARNING: Dynamic Library not loaded", exc_info=True)
```

Comment:
I think only if ASM is enabled, since the appsec span processor will import this file, but then that processor only gets imported if ASM is enabled, correct?

Reply:
If this file was manually imported and the dynamic library was not present, the warning would be shown. We do that  for example in the smoke test to verify that the wheel is properly built.

Otherwise, in the code, tests aside, the only import from `appsec.processor` happens if ASM is enabled